scripts/contentScript.js

```diff
@@ -245,48 +245,26 @@ function addCssElementToBitbucket(highlightedPRIds, userId) {
 }
 
 // adding css elements based up the data getting from api
-function getHighlightedPR(repoOwner, repoName, userId) {
-	chrome.storage.sync.get(["backendUrl"]).then(async ({ backendUrl }) => {
-		const body = {
-			"repo_owner": repoOwner,
-			"repo_name": repoName,
-			"user_id": userId,
-			"is_github": true
-		}
-		const url = `${backendUrl}/relevance/pr`;
-		const highlightedPRIds = await apiCall(url, body);
+function getHighlightedPR(highlightedPRIds) {
```

Comment:
```suggestion
function highlightedRelevantPRs(highlightedPRIds) {
```
Change the name of the function based on the changed actions within it.